docs/eventing/samples/writing-a-source/README.md

```diff
@@ -28,11 +28,11 @@ You'll need these tools installed:
 
 ## Steps
 
-1. [Bootstrap Project](01-bootstrap.md)
-1. [Define The Source Resource](02-define-source.md)
-1. [Reconcile Sources](03-reconcile-sources.md)
-1. [Publish to Cluster](04-publish-to-cluster.md)
-1. Dispatching Events
+1. [Bootstrap Project](./01-bootstrap.md)
+1. [Define The Source Resource](./02-define-source.md)
+1. [Reconcile Sources](./03-reconcile-sources.md)
+1. [Publish to Cluster](./04-publish-to-cluster.md)
+1. (TODO) Dispatching Events
```

Comment:
```suggestion
1. Dispatching Events
```

Reply:
No visible TODOs. Again, could just delete this altogether.

Reply:
see other comment. this clean up work is better in a diff PR since it affects all these files.